src/components/Pricing/data.ts

```diff
@@ -0,0 +1,499 @@
+    },
+    analytics: {
+        label: 'In-product analytics',
+        description: 'Understand user engagement and calculate the value of using Sourcegraph.',
+    },
+}
+
+const DEPLOYMENT_FEATURE_INFO: Record<string, FeatureInfo> = {
+    cloudDeployment: {
+        label: 'Secure and single-tenant Cloud deployment',
+        description: 'Your own dedicated, single-tenant Sourcegraph instance.',
+    },
+    selfDeployment: {
+        label: 'Self-hosted deployment',
+        description: 'Deploy with Docker, Docker Compose, or Kubernetes on your own infrastructure.',
+    },
+}
+
+export const SPOTLIGHT_FEATURE_INFO: Record<string, FeatureInfo> = {
+    ...CIP_FEATURE_INFO,
+
+    // Code host integrations
+    cloudHosts: {
+        label: 'Unlimited standard Cloud code hosts',
+        description: 'Integrate with GitHub.com, GitLab.com, and Bitbucket Cloud.',
+    },
+    unlimitedCodeHosts: {
+        label: 'Unlimited code hosts',
+        description: 'Search, understand, fix, and automate across multiple code hosts.',
+    },
+    selfHostedCodeHosts: {
+        label: 'Connect to self-hosted code hosts',
+        description:
+            'Integrate with Github Enterprise Self-hosted, GitLab Self-hosted, Bitbucket Server/Data Center, and Perforce.',
+    },
+    enterpriseOnlyCodeHosts: {
+        label: 'Connect to enterprise-only code hosts',
+        description: 'Integrate with any Git-based code host via src-srv-git.',
+    },
+    privateCodeHosts: {
+        label: 'Connect to private code hosts',
+        description: 'Integrate with supported code hosts stored behind a firewall (only available for self-hosted deployments).',
+    },
+
+    ...SECURITY_FEATURE_INFO,
+
+    // Scale and performance
+    businessStorage: {
+        label: 'Up to 75GB Cloud code storage',
+        description: 'Included storage for Cloud deployments.',
+    },
+    enterpriseStorage: {
+        label: 'Over 75GB Cloud code storage',
+        description: 'Included storage for Cloud deployments.',
+    },
+    businessExecutors: {
+        label: '2 executors',
+        description:
+            'Offload expensive tasks when running Batch Changes server-side or using code navigation’s auto-indexing functionality.',
+    },
+    enterpriseExecutors: {
+        label: '4 executors',
+        description:
+            'Offload expensive tasks when running Batch Changes server-side or using code navigation’s auto-indexing functionality.',
+    },
+
+    ...DEPLOYMENT_FEATURE_INFO,
+
+    // Support
+    slaSupport: {
+        label: 'Priority support SLAs',
+        description: 'Priority ticket handling and guaranteed initial response SLA from a dedicated team.',
+    },
+    dedicatedManager: {
+        label: 'Dedicated Technical Account Manager',
+        description:
+            'Dedicated technical account manager to support usage, training, enablement, technical strategy, and overall health.',
```

Comment:
Why is "Technical Account Manager" capitalized in the label but not in the description?

Reply:
Is it preferred capitalized or not capitalized? CC: @abschu